requirements.txt

```diff
@@ -4,3 +4,5 @@ openai>=1.30.1,<2
 tiktoken>=0.6,<0.7
 # https://github.com/Yelp/elastalert/issues/2306
 urllib3<2
+# https://github.com/python-pillow/Pillow
+pillow==9.4.0
```

Comment:
this is still not the latest version; is there any particular reason to avoid v10?